<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormOnlinepvp</class>
 <widget class="QWidget" name="FormOnlinepvp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>五子棋——在线对战</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Onlinepvp.qrc">
    <normaloff>:/new/prefix1/Image/Icon/图标.jpg</normaloff>:/new/prefix1/Image/Icon/图标.jpg</iconset>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Background/Gme1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ChessBoard" name="checkerboard_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>840</width>
     <height>840</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Background/check.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>170</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Chess/play2.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player1">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>370</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Chess/play1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="undo_Button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>710</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_Button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>800</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>保存棋局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="defeat_Button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>610</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>认输</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="time_lcdNumber">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>220</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="player_1_label">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>80</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Images/Hint/提示.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player_2_label">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>400</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Images/Hint/提示.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="whosturn_label">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>对方回合</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="player2_name_label">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>190</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>P1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="player1_name_label">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>510</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>P2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChessBoard</class>
   <extends>QLabel</extends>
   <header location="global">chessboard.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Onlinepvp.qrc"/>
 </resources>
 <connections/>
</ui>
